server/test/rating.spec.js

```diff
@@ -10,12 +11,15 @@ describe('Tests API endpoint to rate articles', () => {
    * @description - POST (it should login a valid user)
   */
   before((done) => {
-    const { username, password } = userFaker.validUserDetails;
+    // const { username, password } = userFaker.validUserDetails;
```

Comment:
Please remove this line of code that you've commented out if it is not doing anything useful.

Reply:
done